### DISCUSSION TITLE
Mobile WebView functionality

### DISCUSSION THREAD

**Discussion by devmitch** [DioxusLabs/dioxus#1435] [General]
Hello, in the mobile docs it is mentioned that:
>  WebView doesn't support animations, transparency, and native widgets.

Are there plans to support these features? Or is the WGPU renderer planned to reach parity with the WebView renderer on desktop?

**Comment by ealmloff** [comment]
You can do animations with CSS, and transparency through a config (see the https://github.com/DioxusLabs/dioxus/blob/master/examples/overlay.rs example).
